STYLE_GUIDE.md

```diff
@@ -154,6 +154,27 @@ Prefer:
 
 ## JavaScript
 
+### Variable naming
+
+- Prefer camelCase over snake_case, except in JSON.
+- Try to use verbs for function/method names, and nouns for constant values.
+- Boolean variable names should give some indication of their state.
```

Comment:
Not clear to me what you mean exactly - definitely give some examples - is this like whether it is positive or negative?